### DISCUSSION TITLE
Does anyone else also have Newtonsoft.Json.JsonReaderException when getting Artist?

### DISCUSSION THREAD

**Discussion by rogamaral** [JohnnyCrazy/SpotifyAPI-NET#925] [Q&A]
Since yesterday and suddenly, **without any changes to the code**, I started to have the following exception:

Newtonsoft.Json.JsonReaderException:
'Input string '359120.0' is not a valid integer.
Path 'followers.total',
line 1, position 127.'

`var artist = await _spotify.Artists.Get(artistId);`

Does anyone else also have this problem?

Thanks.

**Comment by martenvill** [comment]
Same here - exactly the same function and no other issues found so far. No time to start digging into the the actual Spotify API requests, but sounds like followers.total value should be an integer, but it is now with a trailing zero for some reason. Perhaps Spotify did some changes in their Json format.

**Comment by janbloemacher** [comment]
This is now also happening to Tracks. Every int is now a float which makes no sense. 

Spotify is doing great implementing changes that nobody asked for and breaks typed applications.

https://community.spotify.com/t5/Spotify-for-Developers/duration-ms-suddenly-returning-float-instead-of-int/td-p/5911044
